src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -95,8 +154,46 @@ const CollectionSelect = ({
     }
   }, [collectionSearchText, collections])
 
+  const itemKey = useCallback((index: number, data: WalletCollection[]) => {
+    if (!data) return index
+    const collection = data[index]
+    return `${collection.address}_${index}`
+  }, [])
+
+  // If there are more items to be loaded then add an extra row to hold a loading indicator.
+  const itemCount = hasNextPage ? displayCollections.length + 1 : displayCollections.length
+
+  // Only load 1 page of items at a time.
+  // Pass an empty callback to InfiniteLoader in case it asks us to load more than once.
+  const loadMoreItems = isFetchingNextPage ? () => null : fetchNextPage
+
+  // Every row is loaded except for our loading indicator row.
+  const isItemLoaded = useCallback(
```

Comment:
Is this different than the `isSuccess` bool passed back from the query?

Reply:
yes, it is checking whether or not each line item is loaded (the input parameter is the line index)